mantidimaging/gui/windows/live_viewer/presenter.py

```diff
@@ -77,8 +78,12 @@ def select_image(self, index: int) -> None:
 
     def load_and_display_image(self, image_path: Path):
         try:
-            with tifffile.TiffFile(image_path) as tif:
-                image_data = tif.asarray()
+            if image_path.__str__().lower().endswith('.tif') or image_path.__str__().lower().endswith('.tiff'):
+                with tifffile.TiffFile(image_path) as tif:
+                    image_data = tif.asarray()
+            elif image_path.__str__().lower().endswith('.fits'):
+                with fits.open(image_path.__str__()) as fit:
+                    image_data = fit[0].data
```

Comment:
`image_path` is of type `Path`, so we can make use of the `suffix` property to get the extension of a file.
I would recommend using the `suffix` property to simplify the if else logic cases like this:
```suggestion
            if image_path.suffix.lower() in [".tif", ".tiff"]:
                with tifffile.TiffFile(image_path) as tif:
                    image_data = tif.asarray()
            elif image_path.suffix.lower() == ".fits":
                with fits.open(image_path.__str__()) as fit:
                    image_data = fit[0].data
```